source/constructs/api/data_source/crud.py

```diff
@@ -504,6 +511,51 @@ def delete_jdbc_connection(provider: str, account: str, region: str, instance: s
     session.commit()
 
 
+def update_jdbc_connection_full(jdbc_instance: schemas.JDBCInstanceSourceUpdate):
+    session = get_session()
+    jdbc_instance_source: JDBCInstanceSource = session.query(JDBCInstanceSource).filter(JDBCInstanceSource.instance_id == jdbc_instance.instance_id,
+                                                                                        JDBCInstanceSource.region == jdbc_instance.region,
+                                                                                        JDBCInstanceSource.account_id == jdbc_instance.account_id,
+                                                                                        JDBCInstanceSource.account_provider_id == jdbc_instance.account_provider_id).order_by(
+        desc(JDBCInstanceSource.detection_history_id)).first()
+    
+    # description: Optional[str]
+    # jdbc_connection_url: Optional[str]
+    # jdbc_enforce_ssl: Optional[str]
+    # kafka_ssl_enabled: Optional[str]
+    # master_username: Optional[str]
+    # password: Optional[str]
+    # skip_custom_jdbc_cert_validation: Optional[str]
+    # custom_jdbc_cert: Optional[str]
+    # custom_jdbc_cert_string: Optional[str]
+    # network_availability_zone: Optional[str]
+    # network_subnet_id: Optional[str]
+    # network_sg_id: Optional[str]
+    # jdbc_driver_class_name: Optional[str]
+    # jdbc_driver_jar_uri: Optional[str]
+
+    jdbc_instance_source.description = jdbc_instance.description
+    jdbc_instance_source.jdbc_connection_url = jdbc_instance.jdbc_connection_url
+    jdbc_instance_source.jdbc_enforce_ssl = jdbc_instance.jdbc_enforce_ssl
+    jdbc_instance_source.kafka_ssl_enabled = jdbc_instance.kafka_ssl_enabled
+    jdbc_instance_source.master_username = jdbc_instance.master_username
+    jdbc_instance_source.skip_custom_jdbc_cert_validation = jdbc_instance.skip_custom_jdbc_cert_validation
+    jdbc_instance_source.custom_jdbc_cert = jdbc_instance.custom_jdbc_cert
+    jdbc_instance_source.custom_jdbc_cert_string = jdbc_instance.custom_jdbc_cert_string
+    jdbc_instance_source.network_availability_zone = jdbc_instance.network_availability_zone
+    jdbc_instance_source.network_subnet_id = jdbc_instance.network_subnet_id
+    jdbc_instance_source.network_sg_id = jdbc_instance.network_sg_id
+    jdbc_instance_source.jdbc_driver_class_name = jdbc_instance.jdbc_driver_class_name
+    jdbc_instance_source.jdbc_driver_jar_uri = jdbc_instance.jdbc_driver_jar_uri
+    jdbc_instance_source.glue_database = None
+    jdbc_instance_source.glue_crawler = None
+    jdbc_instance_source.glue_connection = None
+    jdbc_instance_source.glue_vpc_endpoint = None
+    jdbc_instance_source.glue_crawler_last_updated = datetime.datetime.utcnow()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Naive datetime objects are treated by many datetime methods as local times, We recommend that you create aware timestamps with the `now` or `fromtimestamp` functions using the `tzinfo` to attribute to represent times in UTC. Do not use the [datetime.utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) because it returns only a `naive` datetime object. 

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)